src/modules/apps/e2e/report/failedApps.tsx

```diff
@@ -80,17 +90,28 @@ const ErrorVisualization: React.FunctionComponent<ErrorVisualizationProps> = ({
   error,
   stack,
   testScreenshots,
+  testVideo,
+  specId,
+  testId,
 }) => {
-  const testScreenshotsText = testScreenshots.map(curScreenshot => ` ${curScreenshot.path}`).join('\n')
+  const { account, workspace } = SessionManager.getSingleton()
+  const testScreenshotsUrl = testScreenshots
+    .map(
+      curScreenshot =>
+        `${workspace}--${account}.myvtex.com/_v/screenshot/${testId}/${specId}/${curScreenshot.screenshotId}`
+    )
+    .join('\n')
+  const videoUrl = testVideo === 'true' ? `${workspace}--${account}.myvtex.com/_v/video/${testId}/${specId}` : 'false'
```

Comment:
testVideo is a string with 'false' or 'true'?

Reply:
I think videoUrl can be `''` if testVideo is not 'true'

Reply:
Yes, testVideo is a string with 'false' or 'true'. 
I changed the videoUrl as suggested